TaskLayer/SearchTask/PostSearchAnalysisTask.cs

```diff
@@ -905,6 +909,128 @@ private void WritePeptideResults()
+            {
+                var variants = entry.BestMatchingPeptides.SelectMany(p => p.Peptide.Protein.AppliedSequenceVariations.Where(v => PsmTsvWriter.IntersectsWithVariation(p.Peptide, v, true) ==true).ToList()).ToList();
+                var mods = entry.ModsIdentified;
+                bool modifiedVariant = false;
+                foreach (var mod in mods)
+                {
+                    int residue = mod.Value;                    
+                    var modOnVariant = variants.Where(p => p.OneBasedBeginPosition >= residue && p.OneBasedEndPosition <= residue);
+                    if (modOnVariant.Count() > 0)
+                    {
+                        modifiedVariant = true;
+                    }
+                }
+                if (modifiedVariant == true)
+                {
+                    modifiedVariantSitePeptides.Add(entry);
+                }
+            }
+            foreach (var peptide in confidentVariantPeps)
+            {
+                var variantPWSM = peptide.BestMatchingPeptides.FirstOrDefault();
+                var variants = variantPWSM.Peptide.Protein.AppliedSequenceVariations;
+                foreach (var variant in variants)
+                {
+                    bool intersects = PsmTsvWriter.IntersectsWithVariation(variantPWSM.Peptide, variant, true);
+                    if(intersects == true)
+                    {
+                        if (variant.Description.Description.Contains("missense_variant"))
+                        {
+                            savCount++;
+                        }
+                        if (variant.Description.Description.Contains("frameshift_variant"))
+                        {
+                            frameshiftCount++;
+                        }
+                        if (variant.Description.Description.Contains("stop_gained"))
+                        {
+                            stopGainCount++;
+                        }
+                        if (variant.Description.Description.Contains("conservative_inframe_insertion") || variant.Description.Description.Contains("disruptive_inframe_deletion"))
+                        {
+                            indelCount++;                            
+                        }
+                        
+                    }
+                }
+            }
+
+            string[] variantResults = new string[12];
+            variantResults[0] = "Variant Result Summary";
+            variantResults[2] = "--------------------------------------------------";
+            variantResults[4] = "Number of ambiguous variant peptides at 1% FDR: " + variantPeptides.Count();
+            variantResults[5] = "Number of variant peptides at 1% FDR: " + confidentVariantPeps.Count();
+            variantResults[6] = "Number of SAV variant peptides at 1% FDR: " + savCount;
+            variantResults[7] = "Number of frameshift variant peptides at 1% FDR: " + frameshiftCount;
```

Comment:
I think you can say `frameshift indel peptides`: I don't think there's a way to get a frameshift without an insertion or deletion.

Reply:
Can we separate insertions and deletions in the results?

Reply:
I have separated  inframe insertions and deletions. But I will have to keep framshifts separate in this case because there is no distinction in the description in the frameshift results form an insertion or deletion

Reply:
Can't you check if the applied variant has a longer original or variant sequence?